packages/zeebe-element-templates-json-schema/src/defs/properties.json

```diff
@@ -483,6 +495,121 @@
             }
           }
         }
+      },
+      {
+        "description": "Result variable for script task must not support feel expressions.",
```

Comment:
This shows in the editor even when there is no error in the schema:
![image](https://github.com/user-attachments/assets/f143347d-a8e4-418f-ab86-65aba5fd7f98)

Reply:
How are those descriptions tackled? For example, if we move it inside the `if` are rendered conditionally? Or is it better to get rid of them?

Reply:
We should only use descriptions for describing a positive property (e.g. what a binding is), but never for error messages which are handled via a separate JSON.

Reply:
We could consider using some other JSON schema features to document it, but I'd rather not introduce it in this PR.

Reply:
I'm cleaning up the descriptions.